Attendance.csv

```diff
@@ -0,0 +1,2 @@
+Date,Session,Attendance,Notes,
+2016-12-07,Intro Git,x,,
```

Comment:
oops, I was going to write in the date of the Hack UofT Coders (which isn't even the 7th, it's the 8th). 
D'oh!